src/input_xml.F90

```diff
@@ -5764,10 +5696,10 @@ subroutine assign_sab_tables()
         ASSIGN_SAB: do k = 1, size(mat % i_sab_tables)
           ! In order to know which nuclide the S(a,b) table applies to, we need
           ! to search through the list of nuclides for one which has a matching
-          ! zaid
+          ! name
           associate (sab => sab_tables(mat % i_sab_tables(k)))
             FIND_NUCLIDE: do j = 1, size(mat % nuclide)
-              if (any(sab % zaid == nuclides(mat % nuclide(j)) % zaid)) then
+              if (any(sab % nuclides == nuclides(mat % nuclide(j)) % name)) then
```

Comment:
Hmm, not actually an infinite loop, just very long.  For some reason, `size(sab % nuclides) = 251129904`